044_primenumberinarange.js

```diff
@@ -0,0 +1,14 @@
+let array = [];
+const primeInRange = (arr) => {
+  let max = Math.max(arr[0], arr[1]);
+  let min = Math.min(arr[0], arr[1]);
+  for (let i = min; i <= max; i++) {
+    if (max % i !== 0) {
+      array.push(i);
+    }
+  }
+

```

Comment:
Is this space for readability, adding above for would be better than this. Return statement hast to be placed along with the end brace.